src/libraries/Microsoft.PowerFx.Core/Glue/SimpleResolver.cs

```diff
@@ -48,18 +50,30 @@ internal class SimpleResolver : INameResolver
 
         public IExternalEntity CurrentEntity => null;
 
-        public SimpleResolver(IEnumerable<EnumSymbol> enumSymbols, params TexlFunction[] extraFunctions)
+        /// <summary>
+        /// Initializes a new instance of the <see cref="SimpleResolver"/> class.
+        /// </summary>
+        /// <param name="config"></param>
+        public SimpleResolver(PowerFxConfig config)
```

Comment:
>public SimpleResolver(PowerFxConfig config) [](http://example.com/codeflow?start=8&length=43)

This is the critical ctor. We should just need a config.  If there's anything else, then that we would have passed to the ctor, then it ought to be in the config. 

Remove the other ctors.